frontend/packages/ux-editor/src/containers/DesignView/FormTree/FormItem/FormItemTitle/useDeleteItem.ts

```diff
@@ -0,0 +1,19 @@
+import { useStudioUrlParams } from 'app-shared/hooks/useStudioUrlParams';
+import { useDeleteFormContainerMutation } from '../../../../../hooks/mutations/useDeleteFormContainerMutation';
+import { useDeleteFormComponentMutation } from '../../../../../hooks/mutations/useDeleteFormComponentMutation';
```

Comment:
Could the two mutations ```useDeleteFormContainerMutation``` and ```useDeleteFormComponentMutation``` be moved into an ```index.ts``` file in the ```../hooks/mutations/``` folder so that they can be imported together?

Reply:
I understand the reasoning for doing so, but since the folder exports several things and we are not exporting this as a library package, I'm not in favor of this. It makes refactoring harder and we lose one of the advantages of having the functions in separate files. If we at some point decide to move something out of or into the folder, we will have to update the index file and all the imports manually. Refactoring tools are not that smart - they will just change the reference inside the index file.

Reply:
That makes a lot of sense actually 😄